app/views/userAccount.scala.html

```diff
@@ -73,19 +72,17 @@ <h5>No Passkeys Registered</h5>
                         </div>
                     }                    
                     <div class="section">
-                        <button id="register-passkey" 
-                                class="btn" 
-                                csrf-token="@{CSRF.getToken.get.value}">
+                        <button id="register-passkey" class="btn">
                             Register a new passkey
                         </button>
                     </div>
                 </div>
             </div>
         </div>
     </div>
-    <div id="passkey-name-modal" class="modal">
+    <div id="passkey-name-modal" class="modal" role="dialog" aria-modal="true" aria-labelledby="modal_label" inert>
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/inert for more info about what the `inert` attribute does.